tests/parser/exceptions/test_invalid_type_exception.py

```diff
@@ -30,19 +31,22 @@
 x: [bar(int128), baz(baffle)]
     """,
     """
-x: {bar: int128, decimal: int128}
-    """,
-    """
-x: {bar: int128, 5: int128}
```

Comment:
```
struct X:
  5: int128
```
This is now caught at the parsing stage as a SyntaxError

Reply:
Nice!